source/solvers/navier_stokes_base.cc

```diff
@@ -876,36 +877,15 @@ NavierStokesBase<dim, VectorType, DofsType>::postprocess(bool firstIter)
         }
     }
 
-  // The average_solution and reynolds_stresses vectors are reinitialized at the
-  // first iteration.
-  // Since the reynolds_stress (rs) vector doesn't have the same number of
-  // elements than the number of dofs, it requires its own dof_handler object
-  // even if data is not dofs. It allows to know locally owned elements and
-  // to output data.
   // The average velocities and reynolds stresses are calculated when the
   // time reaches the initial time. (time >= initial time) with 1e-6 as
   // tolerance.
```

Comment:
Maybe the tolerance should be 1e-6 * dt. This way it would be made dimensionless in a way.

Reply:
Ok!